stan/math/prim/scal/prob/frechet_lpdf.hpp

```diff
@@ -60,21 +58,21 @@ typename return_type<T_y, T_shape, T_scale>::type frechet_lpdf(
       log_alpha(length(alpha));
   for (size_t i = 0; i < length(alpha); i++)
     if (include_summand<propto, T_shape>::value)
-      log_alpha[i] = log(value_of(alpha_vec[i]));
+      log_alpha[i] = std::log(value_of(alpha_vec[i]));
```

Comment:
OK, there are a lot of these.  I'm not sure why these were inserted, but I'm pretty sure they're inappropriate.